core/input/devices.js

```diff
@@ -136,10 +136,6 @@
 
             //Util.Debug(">> Keyboard.ungrab");
         },
-
-        sync: function (e) {
-            this._handler.syncModifiers(e);
-        }
```

Comment:
it might not be a bad idea to keep a minimal form of this, where we release all modifiers if there are no modifiers at all pressed -- for embedded noVNC, (where noVNC could lose focus while a key was down), it might be useful, since accidentally pressing `Ctrl-SOMETHING` is probably not the best.